### PR TITLE
Clin 1079 2

### PR DESCRIPTION
**Pull request by czhu-carecloud**
- Moved CC specific files into CC namespace
- Created ICareCloudHapiProperties Interface 
- Committed local .idea configs

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gpabon-carecloud) [inline .project]: These are Eclipse files. They shouldn't go in the project. Are you using Eclipse?

File: .idea/misc.xml [hunk 2]
```diff
@@ -10,5 +10,5 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" project-jdk-name="13.0.2" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="false" project-jdk-name="11" project-jdk-type="JavaSDK" />
```
Comment (gpabon-carecloud) [inline .idea/misc.xml]: Make sure to use JDK v 13 (I use openjdk)

File: hapi-fhir-jpaserver-starter.iml [hunk 3]
```diff
@@ -66,7 +67,7 @@
     <orderEntry type="library" name="Maven: org.eclipse.jetty.websocket:websocket-common:9.4.24.v20191120" level="project" />
     <orderEntry type="library" name="Maven: mysql:mysql-connector-java:8.0.11" level="project" />
     <orderEntry type="library" scope="RUNTIME" name="Maven: com.google.protobuf:protobuf-java:2.6.0" level="project" />
-    <orderEntry type="library" name="Maven: org.postgresql:postgresql:42.2.9" level="project" />
+    <orderEntry type="library" name="Maven: org.postgresql:postgresql:42.2.5" level="project" />
```
Comment (gpabon-carecloud) [inline hapi-fhir-jpaserver-starter.iml]: Any particular reason you are using al older postgresql driver? Don't downgrade it in the dependencies (even if they are just IntelliJ configs)

File: pom.xml [hunk 4]
```diff
@@ -70,7 +70,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>42.2.9</version>
+            <version>42.2.5</version>
```
Comment (gpabon-carecloud) [inline pom.xml]: Downgrade?

File: src/main/java/com/carecloud/fhir/hapi/stu3/ICareCloudHapiProperties.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
+package com.carecloud.fhir.hapi.stu3;
+
+import ca.uhn.fhir.context.FhirVersionEnum;
+import io.github.cdimascio.dotenv.Dotenv;
+
+public interface ICareCloudHapiProperties {
```
Comment (gpabon-carecloud) [inline src/main/java/com/carecloud/fhir/hapi/stu3/ICareCloudHapiProperties.java]: Why is this an interface and not a class? If anything, this should extend the HapiProperties class.

Comment (czhu-carecloud) [reply src/main/java/com/carecloud/fhir/hapi/stu3/ICareCloudHapiProperties.java]: Moving this into a class

File: .project (hunk 1 above)
Comment (czhu-carecloud) [reply .project]: I'm not using Eclipse, not sure how that snuck in there... maybe during my initial config of the project

File: hapi-fhir-jpaserver-starter.iml (hunk 3 above)
Comment (czhu-carecloud) [reply hapi-fhir-jpaserver-starter.iml]: will upgrade drivers

File: .idea/misc.xml (hunk 2 above)
Comment (czhu-carecloud) [reply .idea/misc.xml]: will upgrade to JDK v 13

File: .gitignore [hunk 6]
```diff
@@ -228,3 +228,5 @@ local.properties
 .scala_dependencies
 .worksheet
 
+.env
```
Comment (gpabon-carecloud) [inline .gitignore]: Move this to the top of the file. Before: 

> \#########
> \# DO NOT EDIT PAST THIS POINT
> \# The rest of the rules in this file are generated based on IntelliJ gitignore tool

**Pull request merged**